source/systems/research.md

```diff
@@ -17,15 +17,26 @@ Each column of a research tree is also the level the University needs to be to b
 | 4                    | 4                        |
 | 5 and higher         | 5                        |
 
-You can only choose one column 6 research in each of the Combat, Civilian, and Technology trees.
+You can only have one column 6 research in each of the Combat, Civilian, and Technology trees.  To unlock a different column 6 research for that tree, you must undo the completed one first.
+
+| Symbol | Description |
+| ------ | ----------- |
+| <img src="../../assets/images/gui/university/university_research_blocked.png" alt="Blocked Research"> | Research can be **Blocked**, either by unfinished prerequisite research, by a completed blocking research, or because the research tree has another column 6 research already active. |
```

Comment:
```suggestion
| <img src="../../assets/images/gui/university/university_research_blocked.png" alt="Blocked Research"> | A research can be **blocked**, either by an unfinished prerequisite research, by a completed blocking research, or because the research tree has another column 6 research already active. |
```